src/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const NavBar: React.FC = () => (
+  <nav
+    data-cy="nav"
+    className="navbar is-fixed-top has-shadow"
+    role="navigation"
+    aria-label="main navigation"
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          to="/"
+          className={({ isActive }) => classNames('navbar-item',
+            {
+              'has-background-grey-lighter': isActive,
+            })}
+        >
+          Home
+        </NavLink>
+
+        <NavLink
+          to="/people"
+          className={({ isActive }) => classNames('navbar-item',
+            {
+              'has-background-grey-lighter': isActive,
+            })}
+        >
+          People
+        </NavLink>
```

Comment:
it would be great to create component in this case instead of doing the same thing twice